assignments/#10.1.py

```diff
@@ -0,0 +1,34 @@
+"""
+This program split given large number into hundreds, tens and units.
+"""
+
+
+def split_number(num):
+    result = []
+    i = 10
+    while num > 0:
+        c = num % i
+        num = num - c
+        i = i*10
+        result.append(c)
+    return list(reversed(result))
+
+
+if __name__ == '__main__':
+    print(__doc__)
+
+    test_cases = [
+        (1, [1]),
+        (1025, [1000,0,20,5]),

```

Comment:
Those are test cases. Each list element is a pair of input and expected output.
For example: 'for input `1025` we expect output 1000, 0, 20 and 5'